main.c

```diff
@@ -1457,6 +1457,7 @@ int main(int argc, char **argv) {
 			free(tmp);
 		}
 		memset(cpassword, 0, strlen(cpassword));
+		__asm__ volatile ("" ::: "memory");
```

Comment:
e.g. here this can be `memset_s(cpassword, PASSWORD_BUFSIZE, 0, strlen(cpassword));`